lib/universal-search.js

```diff
@@ -88,14 +88,27 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
+    console.log(osString);
```

Comment:
I think we can remove the console log. Nothing particularly magical here, and we can double-check this value by running `Services.appinfo.OS` directly in the browser toolbox console.